dimod/reference/samplers/exact_solver.py

```diff
@@ -159,6 +161,47 @@ def sample_poly(self, polynomial, **kwargs):
         """
         return ExactSolver().sample(polynomial, **kwargs)
 
+class ExactDQMSolver():
+    """A simple exact solver for testing and debugging code using your local CPU.
+
+    Notes:
+        This solver calculates the energy for every possible
+        combination of variable cases. If variable i has
+        k_i many cases, this will be k_1 * k_2 * ... * k_n
+        which grows exponentially for constant k_i in the
+        number of variables.
+
+    """
+    properties = None
+    parameters = None
+
+    def __init__(self):
+        self.properties = {}
+        self.parameters = {}
+
+    def sample_dqm(self, dqm, **kwargs):
+        """Sample from a discrete quadratic model.
+
+        Args:
+            dqm (:obj:`~dimod.DiscreteQuadraticModel`):
+                Discrete quadratic model to be sampled from.
+
+        Returns:
+            :obj:`~dimod.SampleSet`
+
+        """
+        remove_unknown_kwargs = Sampler.remove_unknown_kwargs(self, **kwargs)
```

Comment:
If you're just calling it in the method, you don't need the LHS. Can just do `Sampler.remove_unknown_kwargs(self, **kwargs)`